support/client/lib/vwf/view/kineticjs.js

```diff
@@ -632,28 +619,26 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color" ],
                     break;
 
                 case "visible":
-                    if ( pauseRendering ) {
+                    var visible = propertyValue;
+                    if ( this.state.pauseRendering ) {
                         drawThis = false;
-                    }
-                    else {
+                    } else {
                         drawThis = !activelyDrawing;
-                        if ( !propertyValue ) {
-                            clearBefore = !propertyValue;
-                        }
+                        clearBefore = visible ? clearBefore : false;
                     }
                     break;
 
                 case "pauseRendering":
+                    this.state.pauseRendering = propertyValue;
+                    drawThis = false;
```

Comment:
The relationship in other places between these values seems to be:

    drawThis = !pauseRendering && !activelyDrawing

Should `drawThis` always be set `false`, even if `pauseRendering` is being set `false`? Or are the other states factored out in this case?